lib/run.tsx

```diff
@@ -43,6 +43,7 @@ const makeUseMDXComponents = (more: ReturnType<UseMdxComponents> = {}): UseMdxCo
     'embed-block': Components.Embed,
     'image-block': Components.Image,
     'table-of-contents': Components.TableOfContents,
+    'cards-grid': Components.CardsGrid,
```

Comment:
I think you don't need this line. I actually think the preceding 3 or 4 are unnecessary too, but I can follow up on that. These are the tag names, so `CardsGrid` should already be getting pulled in here: https://github.com/readmeio/markdown/blob/c683a805986877f3f029176616d5b5607b0ce240/lib/run.tsx#L35

This has me wondering if overwriting, for instance, the `img` tag is a good idea. This makes it impossible for someone to use them in a custom component without our component:

```
const CustomImage = (props) => {
  return <img {...props} />
}
```